tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/connect/AMQPFederationAddressPolicyTest.java

```diff
@@ -3882,6 +3883,300 @@ public void testRemoteFederationReceiverCloseWithErrorTerminateRemoteConnection(
       }
    }
 
+   @Test
+   @Timeout(20)
+   public void testRemoteReceiverClosedWhenDemandRemovedCleansUpAddressBinding() throws Exception {
+      server.start();
+      server.addAddressInfo(new AddressInfo(SimpleString.of("test"), RoutingType.MULTICAST));
```

Comment:
Rather than literal "test" could it use e.g getTestName()? Perhaps assigned to a variable to make following the address usage simpler.

Reply:
I prefer it the way it is, plus it falls inline with the remainder of the tests.

Reply:
I prefer tests use specific names as it makes debugging them later amongst a sea of logs easier, and finding cross-test pollutions easier, and helps avoid them having an impact to begin with. I prefer variables to repeated literals because you can follow their usage with IDEs and update them more easily if needed later.